binderhub/app.py

```diff
@@ -98,6 +110,41 @@ def _valid_base_url(self, proposal):
         config=True
     )
 
+    appendix = Unicode(
+        help="""
+        Appendix to pass to repo2docker
+
+        A multi-line string of Docker directives to run.
+        Since the build context cannot be affected,
+        ADD will typically not be useful.
+
+        This should be a Python string template.
+        It will be formatted with at least the following names available:
+
+        - binder_url: the binder URL for (e.g. for linking)
+        - repo_url: the repository URL used to build the image
+        """,
+        config=True,
+    )
+
+    binderhub_appendix_url = Unicode(
+        help="""url of binderhub git-archive tarball for using the default appendix
```

Comment:
"URL of the git-archive tarball to use as the default appendix"?

Why default? The user/repo creator can't change this so this is _the_ appendix for this binderhub instance no?

I'd change L133 to "the contents of `appendix/` from the Binderhub repository will be used if this is not set"